### DISCUSSION TITLE
Ready for the 1.1.13 release ?

### DISCUSSION THREAD

**Discussion by bcoconni** [JSBSim-Team/jsbsim#772] [General]
I am about to push the button to release `v1.1.13`. I have compiled the changelog below.
Is there anything important that is missing or should be included in this release ?

### User
* The license of *libexpat* is now included in the Windows installer and in the Python wheels.
* Input sockets that were broken on Windows have been fixed.
### Python module
* The Python script `JSBSim` has been renamed `JSBSim.py` and now has the same features than the binary executable `JSBSim.exe`.
* Fixed the data folders that are installed along with the Python module: some aircraft sub-directories `System` and `Engines` were missing in the previous release (issue #687)
* Python wheels for MacOSX are now provided as *universal* binaries for Python `>=3.8` which allows using these binary wheels with Apple Silicon chips.
* Python wheels for Python 3.11 are now available.
* JSBSim Python module is now installed under a single folder including the licenses and the aircraft data. In the previous release, the files were scattered among several folders. This allows a cleaner installation and removal (PR #755).
* Type hints are now provided along with JSBSim Python module (as per PEP 561 and PR #756).
* Usage of uninitialized instances of JSBSim classes now throws an exception to avoid a crash of the Python interpreter.
* `eTemperature` and `ePressure` are now derived from the Python module `enum`.
* Some internal methods are no longer accessible. Their names were prepended by the double underscore `__` which according to Python programming guidelines are meant to be private methods. So you were warned !
### MATLAB S-Function
* By default, the S-Function is now compiled using the flag `JSBSIM_STATIC_LINK` since JSBSim is supplied as a static library in the Windows installer (PR #735).
### Developer
* Removed the `Messages` feature as it was unused and was using a `static` member that could lead to a segmentation fault when using several instances of `FGFDMExec` (issue #666)
* Restored the automatic rebuild of the Python module that should occur when the headers included in `jsbsim.pyx` are modified.
* Replaced `CMAKE_SOURCE_DIR` by `PROJECT_SOURCE_DIR` to allow JSBSim being built using CMAKE FetchContent (issue #684)
* The dependency to the `win32api` module has been removed to build JSBSim Python module on Windows.
* Random number generation is now made using C++11 standard library. This allowed removing another `static` member (issue #666)
* Fixed a bug that prevented building JSBSim as a DLL using the MinGW compiler (issue #719)
* Unit tests have been added for the following classes: `FGRealValue`, `FGParameter`, `FGParameterValue`, `FGCondition`.

**Comment by seanmcleod** [comment]
Looks good, can't thing of anything of anything off-hand that might be missing.

**Reply by bcoconni** [reply]
Thanks v1.1.13 has just been released.
